code/modules/mob/emote.dm

```diff
@@ -83,3 +83,21 @@
 						riding_datum.force_dismount(M)
 			else
 				L.unbuckle_all_mobs()
+
+/datum/emote/spin/check_cooldown(mob/user, intentional)
```

Comment:
You can probably just make this `mob/living/carbon/user`, then check `if (!istype(user)) return` to avoid making the spinner variable completely.